she_codes_news/news/templates/news/story.html

```diff
@@ -1,8 +1,35 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block stylesheets %}
+<link rel='stylesheet' type='text/css' href="{% static 'news/styles.css' %}">
+{% endblock %} 
+
+{% block content %}
+
 <div id="story-wrapper">
     <h2>{{ story.title }}</h2>
-    <h4>{{ story.author }}</h4>
     <div id="story-content">
-        <img src="{% static '/news/images/placeholder-5.jpg' %}">
+        <img src="{{ story.image }}">
```

Comment:
if image is allowed to be blank, you might want to check here for if the story.image has content before using it as a image